Shared/FeatureFlags/FeatureFlags.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  FeatureFlags.swift
+//  Lockbox
+//
+//  Created by Kayla Galway on 11/8/19.
+//  Copyright © 2019 Mozilla. All rights reserved.
+//
+
+import Foundation
+
+/// Helper struct that checks if Build Configuration is Release or Debug
+struct FeatureFlagSupport {
+    
+    static var isDebug: Bool {
+        #if DEBUG
+            return true
+        #else
+            return false
+        #endif
+    }
+    
+    static var isRelease: Bool {
+        #if RELEASE
+            return true
+        #else
+            return false
+        #endif
+    }
+    
+}
+
+/// Boolean flags that identify if test feature should be shown depending on whether the Build Configuration is Release or Debug
+struct FeatureFlags {
+    
+    /// Feature flag for the ability to edit a credential
+    static var crudEdit: Bool {
+        return FeatureFlagSupport.isDebug ? true : false
```

Comment:
I'm stuck here: do we make this simpler

e.g. 

```swift
static let crudEdit: Bool = FeatureFlagSupport.isDebug
```

or more verbose

```swift
static var crudEdit: Bool {
   switch {
       case FeatureFlag.isDebug: return true
       case FeatureFlag.isRelease: return false
   }

   return false
}
```

Having written this out, I think I'd prefer the latter as it's easier to change, and gives a good basis to copy from.

Reply:
You can't switch over the values of `FeatureFlagSupport` because it isn't an enum, so I used if statements to make it more verbose like your example.